frontend/src/components/associations/library/management/AssociationLibraryManagement.tsx

```diff
@@ -52,30 +52,64 @@ const columnsDefinition = [
     ),
   },
   {
-    key: "action",
-    header: "Action",
+    key: "actions",
+    header: "Actions",
     cellClassName: "text-center",
     render: (loanable: Loanable) => (
-      <OverlayTrigger
-        placement={"bottom"}
-        overlay={<Tooltip id="edit">Modifier</Tooltip>}
-        popperConfig={{
-          modifiers: [
-            {
-              name: "offset",
-              options: {
-                offset: [0, 8],
+      <>
+        <OverlayTrigger
+          placement={"bottom"}
+          overlay={<Tooltip id="edit">Gérer les demandes</Tooltip>}
```

Comment:
`Gérer les demandes` should have its own sidebar link

Reply:
It's specific to a Loanable, so that's not possible